prinz/src/main/scala/pl/touk/nussknacker/prinz/util/collection/immutable/MapMultiset.scala

```diff
@@ -0,0 +1,132 @@
+package pl.touk.nussknacker.prinz.util.collection.immutable
+
+import scala.collection._
+
+object MapMultiset {
+
+  def empty[T](): MapMultiset[T] =
```

Comment:
do we have to use parentheses for functions with no args? we need to talk about these cases at team meeting